src/EnergyPlus/MixedAir.cc

```diff
@@ -1566,100 +1553,103 @@ void GetOAControllerInputs(EnergyPlusData &state)
             // Loop over zones and fill OA and AD specs, if none were found, use defaults
             for (int ventMechZoneNum = 1; ventMechZoneNum <= MechVentZoneCount; ++ventMechZoneNum) {
                 auto &thisVentMechZone = thisVentilationMechanical.VentMechZone(ventMechZoneNum);
-                int zoneOAReqObjIndex = thisVentMechZone.ZoneDesignSpecOAObjIndex;
-                if (zoneOAReqObjIndex > 0) {
-                    auto const &curOARequirements(state.dataSize->OARequirements(zoneOAReqObjIndex));
-                    thisVentMechZone.ZoneOAAreaRate = curOARequirements.OAFlowPerArea;
-                    thisVentMechZone.ZoneOAPeopleRate = curOARequirements.OAFlowPerPerson;
-                    thisVentMechZone.ZoneOAFlowRate = curOARequirements.OAFlowPerZone;
-                    thisVentMechZone.ZoneOAACHRate = curOARequirements.OAFlowACH;
-                    thisVentMechZone.ZoneOAFlowMethod = curOARequirements.OAFlowMethod;
-                    thisVentMechZone.zoneOASched = curOARequirements.oaFlowFracSched;
-                    thisVentMechZone.oaPropCtlMinRateSched = curOARequirements.oaPropCtlMinRateSched;
```

Comment:
No longer need to copy values into the mech vent controller, just access everything from the `ZoneDesignSpecOAObjIndex`.